packages/onramp-kit/src/packs/monerium/SafeMoneriumClient.ts

```diff
@@ -195,31 +190,23 @@ export class SafeMoneriumClient extends MoneriumClient {
    */
   async #isValidSignature(safeAddress: string, messageHash: string): Promise<boolean> {
     try {
-      const eip1271data = EIP_1271_INTERFACE.encodeFunctionData('isValidSignature', [
-        messageHash,
-        '0x'
-      ])
-      const msgBytes = getBytes(messageHash)
-
-      const eip1271BytesData = EIP_1271_BYTES_INTERFACE.encodeFunctionData('isValidSignature', [
-        msgBytes,
-        '0x'
-      ])
-
-      const checks = [
-        this.#safeProvider.call({
-          from: safeAddress,
-          to: safeAddress,
-          data: eip1271data
-        }),
-        this.#safeProvider.call({
-          from: safeAddress,
-          to: safeAddress,
-          data: eip1271BytesData
+      const [eip1271data, eip1271BytesData] = [EIP_1271_ABI, EIP_1271_BYTES_ABI].map((abi) =>
+        encodeFunctionData({
+          abi,
+          functionName: 'isValidSignature',
+          args: [messageHash as Hash, '0x']
```

Comment:
One of the `encodeFunctionData` received the `messageHash`, and the other one received the `msgBytes`.

This simplification can't be done.

Reply:
They should be the same. The previous code had a `getBytes` which should the same hash but in bytes. In abitype both `adressType` and `bytesType` have the same types which is a `0x`-prefixed `string`: https://github.com/wevm/abitype/blob/main/packages/abitype/src/register.ts#L132 which matches the viem type for Hash https://github.com/wevm/viem/blob/f43388495e6f87147108ea101b6083213c5699c0/src/types/misc.ts#L5. I do think its a bit misleading and `Hex` is probably a bit more accurate. I will also test out because surprises happen.